src/FSharpLint.Core/Rules/Conventions/Naming/NamingHelper.fs

```diff
@@ -363,3 +363,19 @@ let rec identFromSimplePat = function
     | SynSimplePat.Id(ident, _, _, _, _, _) -> Some(ident)
     | SynSimplePat.Typed(p, _, _) -> identFromSimplePat p
     | SynSimplePat.Attrib(_) -> None
+
+let rec isNested args nodeIndex =
+    let parent = args.SyntaxArray.[nodeIndex].ParentIndex
+    let actual = args.SyntaxArray.[parent].Actual
+
+    match actual with
+    | AstNode.Expression (SynExpr.LetOrUse (_, _, _, _, _)) -> true
+    | _ -> false
+
+let getFunctionIdents (_accessibility:Accessibility) (pattern:SynPat) =
```

Comment:
@webwarrior-ws if _accessibility param is not used here, why not just remove it?

Reply:
Because `getFunctionIdents` is passed into another function, which expects `Accessibility -> SynPat -> 't` type.

Reply:
And that function is used in multiple places

Reply:
I have an idea, you can remove the _accessibility param here, and when using getFunctionIdents you can use it this way:

```
getPatternIdents Accessibility.Public (fun _accessibility pattern -> getFunctionIdents pattern) true pattern
```

BTW, to make it more understandable that Accessibility.Public is not used, store it first in a variable called `_dummyUnusedAccessibility`, ok?